timeseries/src/autogluon/timeseries/models/multi_window/multi_window_model.py

```diff
@@ -111,11 +110,15 @@ def _fit(
 
         oof_predictions_per_window = []
         global_fit_start_time = time.time()
+        model: Optional[AbstractTimeSeriesModel] = None
 
         for window_index, (train_fold, val_fold) in enumerate(val_splitter.split(train_data)):
             logger.debug(f"\tWindow {window_index}")
+
             # refit_this_window is always True for the 0th window
             refit_this_window = window_index % refit_every_n_windows == 0
+            assert not window_index == 0 or refit_this_window
```

Comment:
Maybe 
```python
assert window_index != 0 or refit_this_window
```
for better readability? I keep forgetting the order of `not` with other operations